### PR TITLE
CIVDT-80: Added Acceptance Date page

### PR DESCRIPTION
**Pull request by dannybassi**

### CODE REVIEW

File: conf/app.routes [hunk 1]
```diff
@@ -17,7 +17,12 @@ POST        /who-is-disclosing               controllers.UserTypeController.onSu
 GET         /one-or-more-entries             controllers.NumberOfEntriesController.onLoad
 POST        /one-or-more-entries             controllers.NumberOfEntriesController.onSubmit
 
+GET         /unauthorised          controllers.errors.UnauthorisedController.onPageLoad
```
Comment (ian-ralphs) [inline conf/app.routes:20]: Can you sort out formatting in this file please?

File: app/views/AcceptanceDateView.scala.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    @layout(pageTitle = messages("acceptanceDate.pageTitle"), form = Some(form), customBackLinkUrl = Some(controllers.routes.EntryDetailsController.onLoad)) {
+
+        @errorSummary(form.errors)
+
+        @formHelper(action = controllers.routes.AcceptanceDateController.onSubmit()) {
+            @yesNoRadio(
+                form,
+                headingKey = "acceptanceDate.heading",
+                inline = false,
+                yesRadioMsg = "acceptanceDate.yes",
```
Comment (ian-ralphs) [inline app/views/AcceptanceDateView.scala.html:34]: Why are you overriding the default Yes No descriptions?

File: test/assets/messages/AcceptanceDateMessages.scala [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+ * limitations under the License.
+ */
+
+package messages
+
+object AcceptanceDateMessages extends BaseMessages {
+
+  val title: String = "Was the entry acceptance date before 1 January 2021?"
+  val h1: String = "Was the entry acceptance date before 1 January 2021?"
+  val radioButtonOne: String = "Yes"
```
Comment (ian-ralphs) [inline test/assets/messages/AcceptanceDateMessages.scala:23]: These will become site.yes and site.no so might be worth moving to the Base Messages file.

**Review by ian-ralphs — CHANGES_REQUESTED** [review]

File: app/views/AcceptanceDateView.scala.html (hunk 2 above)
Comment (dannybassi) [reply app/views/AcceptanceDateView.scala.html:34]: removed yesRadioMsg and noRadioMsg

File: test/assets/messages/AcceptanceDateMessages.scala (hunk 3 above)
Comment (dannybassi) [reply test/assets/messages/AcceptanceDateMessages.scala:23]: removed acceptanceDate.yes and acceptanceDate.no from Base Messages file and added site.yes and site.no

File: conf/app.routes (hunk 1 above)
Comment (dannybassi) [reply conf/app.routes:20]: done

**Pull request merged**